divide_and_conquer/convex_hull.py

```diff
@@ -411,6 +411,77 @@ def _construct_hull(points, left, right, convex_set):
             _construct_hull(candidate_points, extreme_point, right, convex_set)
 
 
+def convex_hull_melkman(points):
```

Comment:
Maybe specify the type for points? You may see our typing guidelines.

Reply:
You may also do this for other functions in this file.

Reply:
done